src/components/Details/MoviesDetails.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { CallMoviesApi } from '../../redux/fetch-movies-titles';
+import MoviesDetailItems from './MoviesDetailItems';
+import './detailpage.css';
+
+const MoviesDetails = () => {
+  const dispatch = useDispatch();
+  const moviesStat1 = useSelector((state) => state.movieDetails.data);
+  const moviesStat = [moviesStat1];
+  useEffect(() => {
+    dispatch(CallMoviesApi());
+  },
+  [dispatch]);
+  return (
+    <div className="detail-parent">
+      <div className="detail-items">
+        {moviesStat.map((detail) => (
+          <div key={uuidv4()}>
+            <MoviesDetailItems
+              title={detail.title}
+              plotoverview={detail.plot_overview}
+              runtimeminutes={detail.runtime_minutes}
+              type={detail.type}
+              userrating={detail.user_rating}
+              poster={detail.poster}
+              trailer={detail.trailer}
+              releasedate={detail.release_date}
+              imdbid={detail.imdb_id}
+              usrating={detail.us_rating}
+              year={detail.year}
+              originallanguage={detail.original_language}
+
+            />
```

Comment:
- [ ] You should not use a fixed width value like `width: 800px;` on the MoviesDetails page instead you could use an example (`%, vw, wh, ...) this will prevent you from having this black white space you have on the left and right, I suggest you change this 👍🏻 
![image](https://user-images.githubusercontent.com/79058364/217823438-c8f469ad-1d4e-4656-9d04-c75fb54b01e3.png)

- [ ] Kindly improve the design of you MoviesDetails page to match the original one 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/217828693-b49d57ec-1cb0-48d3-83e8-743823b60f19.png)